### ISSUE TITLE
DropURL length should be fixed

### ISSUE THREAD

**Issue by audax** [Qabel/qabel.github.io#151]
By allowing the user to chose the length of the DropURL, we reduce the chance for collisions and we have to explain what all that stuff actually means.

Is it really worth it or should we just use a reasonable default value?

**Comment by roeslpa** [comment]
I am in favor of letting the user decide. Because users who use a mobile device would decide different than users who only use the desktop version. But instead of "less" and "much" I would write "little traffic and low anonymity" and "much traffic and high anonymity" and add a description for interested users below.

**Comment by dannywerner** [comment]
yes a description is imho needed. maybe a ''button" until the user can change to traffic.... 'whats this' or 'more infos'. iam send you @audax  a pm with a wording example

**Comment by roeslpa** [comment]
In my current android version (from last Thursday) the description is actually wrong: questions: "how paranoid are you" and "how much traffic would you accept" -> answer: "less" should mean that many URL bits are used (=> few collisions => few users use one URL => little traffic) but it is implemented the other way around: "less" indicates few bits are used.

**Comment by jmt85** [comment]
what is the state of this ticket

**Comment by roeslpa** [comment]
From my point of view if we fix the length to a certain value we (have to) answer the big question nobody ever answered regarding Qabel Drop: (when) **will Qabel Drop scale** (on mobile devices)**?**
If anybody can answer this, please set a default value. I belief nobody can answer this question because of the variable setting of mobile devices. Hence I would like to variably find out the answer by observing the behaviour (at least during the beta).
